spec/javascripts/services/alerts_center_service_spec.js

```diff
@@ -0,0 +1,186 @@
+      expect(sortFields[2].title).toBe(__("Host Name"));
+      expect(sortFields[3].title).toBe(__("Provider Name"));
+      expect(sortFields[4].title).toBe(__("Provider Type"));
+      expect(sortFields[5].title).toBe(__("Owner"));
+      expect(sortFields[6].title).toBe(__("Acknowledged"));
+    });
+
+    it('should give the correct menu actions', function() {
+      var menuActions = testService.menuActions;
+      expect(menuActions.length).toBe(5);
+
+      expect(menuActions[0].name).toBe(__("Acknowledge"));
+      expect(menuActions[1].name).toBe(__("Add Note"));
+      expect(menuActions[2].name).toBe(__("Assign"));
+      expect(menuActions[3].name).toBe(__("Unacknowledge"));
+      expect(menuActions[4].name).toBe(__("Unassign"));
+    });
+  });
+
+  describe('update commands', function() {
+
+    beforeEach(function() {
+      deferred = $q.defer();
+
+      spyOn(API, 'get').and.callFake(function() {return deferred.promise;});
+    });
+
+    it('should get the current user correctly', function() {
+      testService.getCurrentUser();
+
+      deferred.resolve(adminResponse);
+      $rootScope.$apply();
+
+      expect(testService.currentUser.userid).toBe('admin');
+    });
+
+    it('should update existing user correctly', function() {
+      testService.currentUser = adminResponse.identity;
+
+      testService.updateExistingUsers();
+
+      deferred.resolve(existingUsersResponse);
+      $rootScope.$apply();
+
+      expect(testService.existingUsers.length).toBe(2);
+    });
+
+    it('should update existing providers correctly', function() {
+      testService.updateProviders();
+
+      deferred.resolve(providersResponse);
+      $rootScope.$apply();
+
+      expect(testService.providers.length).toBe(4);
+    });
+
+    it('should update existing tags correctly', function() {
+
+      testService.updateTags();
+
+      deferred.resolve(tagsResponse);
+      $rootScope.$apply();
+
+      expect(testService.tags.length).toBe(4);
+    });
+
+    it('should retrieve existing alerts correctly', function() {
+      var alerts = [];
+
+      testService.getAlertsData().then(function(response) {
+        alerts = response.resources;
+        return true;
+      });
+
+      deferred.resolve(alertsResponse);
+      $rootScope.$apply();
+
+      expect(alerts.length).toBe(4);
+    });
+  });
+
+  describe('convert functions', function() {
+    beforeEach(function() {
+      testService.currentUser = adminResponse.identity;
+      testService.existingUsers = existingUsersResponse.resources;
+      testService.providers = providersResponse.resources;
+      testService.tags = tagsResponse.resources;
+    });
+
+    it('should convert into an alert list correctly', function() {
+      var alertsList = testService.convertToAlertsList(alertsResponse);
+
+      expect(alertsList.length).toBe(4);
+
+      expect(alertsList[0].objectType).toBe("Hawkular");
+      expect(alertsList[0].objectTypeImg).toBe("/assets/svg/vendor-hawkular.svg");
```

Comment:
won't be, you may have more luck with something like `toMatch(/svg\/vendor-hawkular/)`